spec/models/comment_spec.rb

```diff
@@ -37,4 +42,73 @@
     comment = build(:comment, markeddown_comment: "a" * 16_777_216)
     expect(comment).to_not be_valid
   end
+
+  it "sends reply notification" do
+    # The user receiving the notification.
+    u = create(:user)
+    u.settings["email_notifications"] = true
+    u.settings["email_replies"] = true
+
+    # The user sending the notification.
+    u2 = create(:user)
+    # Story under which the comments are posted.
+    s = create(:story)
+
+    # Parent comment.
+    c = build(:comment, story: s, user: u)
```

Comment:
```suggestion
    c = create(:comment, user: recipient)
```

Then there's no need for the explicit `c.save` on the next line. Same for next comment.

Reply:
Save as above, isn't it required to share a story object if you're replying to a comment?